README.md

```diff
@@ -1 +1,15 @@
-# tool-template
\ No newline at end of file
+#  BIOPET tool suite
+This tool is part of BIOPET tool suite that is developed at LUMC by [the SASC team](http://sasc.lumc.nl/). Each tool in the BIOPET tool suite is meant to offer a standalone function that can be used to perform a dedicate data analysis task or added as part of [BIOPET pipelines](http://biopet-docs.readthedocs.io/en/latest/).
+
+#  About this tool
+<Description of a tool>
+
+#  Installation
+<Dependency and process to compile a tool>
```

Comment:
Installation is only download of a jar, compiling should not be required for users.

Reply:
General dependency for all tools is java 8